### PR TITLE
stolostron/backlog#20569 hub cluster network

### PR DESCRIPTION
**Pull request by swopebe**
--working

### CODE REVIEW

File: networking/network_config_hub.adoc [hunk 1]
```diff
@@ -1,56 +1,63 @@
...
+
 |===
-|Direction | Protocol | Connection | Port (if specified) | Source address | Destination address
+| Direction | Protocol | Connection | Port (if specified) | Source address | Destination address
 
-| Outbound to managed cluster 
+| Outbound to the managed cluster 
 | HTTPS 
-| Retrieval of logs dynamically from Search console for the pods of the managed cluster. This connection uses a service running on the managed cluster called - klusterlet-addon-workmgr 
+| Retrieval of logs dynamically from Search console for the pods of the managed cluster, uses a service that is running on the managed cluster, the `klusterlet-addon-workmgr`
```
Comment (cadawson) [inline networking/network_config_hub.adoc]: This cell is a little confusing to me. It looks like you are avoiding having a sentence in there, for parallelism. 

If I am not misunderstanding, maybe something like: 
Retrieval of logs dynamically from Search console for the pods of the managed cluster, which uses the `klusterlet-addon-workmgr` service that is running on the managed cluster

**Review by cadawson — CHANGES_REQUESTED** [review]
One suggestion, but otherwise looks good. Thanks!

File: networking/network_config_hub.adoc (hunk 1 above)
Comment (swopebe) [reply networking/network_config_hub.adoc]: It's two different concepts, but I will check with developers to see how best to present it without a run-on sentence.

Comment (swopebe) [reply networking/network_config_hub.adoc]: Sending of logs dynamically from Search console for the pods of the managed cluster, uses the `klusterlet-addon-workmgr` service that is running on the managed cluster

Comment (cadawson) [reply networking/network_config_hub.adoc]: Better than the original, but looks like 2 list items joined by a comma. Maybe split them into 2 list items? Don't spend much time on it. If you are good with it, then roll with it. It's a challenge.

Comment (swopebe) [reply networking/network_config_hub.adoc]: listing in a cell I am against unless absolutely necessary. It is exactly two items separated by a comma, which we do sometimes. Similar to a csv.

**Review by cadawson — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**